src/test/java/uk/gov/pay/directdebit/notifications/services/UserNotificationServiceTest.java

```diff
@@ -85,32 +88,59 @@ public void setUp() {
     }
 
     @Test
-    public void shouldSendEmailIfEmailNotifyIsEnabled() throws Exception {
+    public void shouldSendMandateFailedEmailIfEmailNotifyIsEnabled() throws Exception {
         when(mockNotifyClientFactory.isEmailNotifyEnabled()).thenReturn(true);
         when(mockNotifyClient.sendEmail(any(), any(), any(), any())).thenReturn(mockNotificationCreatedResponse);
 
         userNotificationService = new UserNotificationService(mockDirectDebitConfig, mockNotifyClient, mockMetricRegistry, mockGatewayAccountService);
-        Future<Optional<String>> idF = userNotificationService.sendMandateFailedEmail(payer, transaction);
+        Future<Optional<String>> idF = userNotificationService.sendMandateFailedEmailFor(transaction, payer);
         idF.get(1000, TimeUnit.SECONDS);
 
-        HashMap<String, String> map = new HashMap<>();
-        map.put("org name", gatewayAccount.getServiceName());
-        map.put("org phone", "000-CAKE-000");
-        map.put("dd guarantee link", "https://frontend.url.test/direct-debit-guarantee");
+        HashMap<String, String> emailPersonalisation = new HashMap<>();
+        emailPersonalisation.put("org name", gatewayAccount.getServiceName());
+        emailPersonalisation.put("org phone", "000-CAKE-000");
+        emailPersonalisation.put("dd guarantee link", "https://frontend.url.test/direct-debit-guarantee");
         verify(mockNotifyClient).sendEmail(
-                mockNotifyClientFactory.getMandateFailedTemplateId(),
+                MANDATE_FAILED_TEMPLATE,
                 EMAIL,
-                map,
+                emailPersonalisation,
                 null
         );
     }
 
+    @Test
+    public void shouldSendPaymentConfirmedEmailIfEmailNotifyIsEnabled() throws Exception {
+        when(mockNotifyClientFactory.isEmailNotifyEnabled()).thenReturn(true);
+        when(mockNotifyClient.sendEmail(any(), any(), any(), any())).thenReturn(mockNotificationCreatedResponse);
```

Comment:
We actually have all the information we need to stub this with precise arguments